app/Models/Camp.php

```diff
@@ -453,7 +453,11 @@ public static function getDirectCampSupporterIds($topicid, $campnum)
     public static function validateParentsupport($topicNum, $campNum, $userNicknames) 
     {
         $filter = self::getLiveCampFilter($topicNum, $campNum);
-        $oneCamp = self::getLiveCamp($filter);        
+        $oneCamp = self::getLiveCamp($filter);   
+
+        if(empty($oneCamp)){
```

Comment:
`if(!$onecamp)` will ot work

Reply:
Not sure because sometimes it gives empty eloquent model which does not work  with !